blueprints/showcase-plugin-with-media/blueprint.json

```diff
@@ -0,0 +1,149 @@
+{
+	"$schema": "https://playground.wordpress.net/blueprint-schema.json",
+	"preferredVersions": {
+		"php": "8.3",
+		"wp": "6.5"
+	},
+	"features": {
+		"networking": true
+	},
+	"phpExtensionBundles": [
+		"kitchen-sink"
+	],
+	"landingPage": "/",
+	"steps": [
+		{
+			"step": "login",
+			"username": "admin",
+			"password": "password"
+		},
+		{
+			"step": "installPlugin",
+			"pluginZipFile": {
+				"resource": "wordpress.org/plugins",
+				"slug": "makeiteasy-back-to-top"
+			},
+			"options": {
+				"activate": true
+			}
+		},
+		{
+			"step": "installPlugin",
+			"pluginZipFile": {
+				"resource": "url",
+				"url": "https://makeiteasypopup.makeiteasy.hr/makeiteasy-popup.zip"
```

Comment:
Would it be possible to use paths from this GitHub repo or at least WordPress.org?

In case something changes with the domain, we would still like for the blueprint to work.

Reply:
Ok, I'll rewrite blueprint as soon as I can.